### DISCUSSION TITLE
Why there is problem in Mobile in some cases

### DISCUSSION THREAD

**Discussion by Avi-ADAM** [janosh/svelte-multiselect#292] [Q&A]
I am using svelte-multiselect for long and it was a great help With great features.
Recently in some places on my website I noted that on mobile it is not working fine.
When users clicks on the opened list its jumps out and not selecting the clicked option.
Its only in some places and only in Mobile.
What May cause the problem? I already Tried Many options With z-index..

**Comment by janosh** [comment]
hi there, is this still an issue? if so, could you post a repro with a link and/or some steps to observe the issue?

**Reply by Avi-ADAM** [reply]
I still didnt find the cause of it. 
The page where the problem appers is only for registered users so I provided a video, you can see that only in the end When I close the keyboard I mannaged to choose. 

https://github.com/user-attachments/assets/1f09370c-aff7-4409-a79d-f72dc8e4cb90

This is the code for this component, (the page is in routs/reg/moach) 
https://github.com/Avi-ADAM/1.0/blob/main/src/lib/components/prPr/mission.svelte
